spec/mongoid/config_spec.rb

```diff
@@ -233,6 +233,12 @@
       end
     end
 
+    around do |ex|
+      temp = Mongoid.send(option)
+      ex.run
+      Mongoid.send("#{option}=", temp)
```

Comment:
I would write a macro similar to the one we have for Mongoid options.

Reply:
That would be almost the same as config_override. The difference in this case is that the Mongoid options are being modified in other ways within the tests (using loaded configs), and I just want to put it back to what it was before the test.